static/app/components/nav/config.tsx

```diff
@@ -0,0 +1,192 @@
+            to: `/traces/`,
+            check: {features: 'performance-trace-explorer'},
+          },
+          {
+            label: t('Metrics'),
+            to: `/${prefix}/metrics/`,
+            check: {features: 'custom-metrics'},
+          },
+          {
+            label: t('Profiles'),
+            to: `/${prefix}/profiling/`,
+            check: {features: 'profiling', hook: 'profiling-sidebar-item'},
+          },
+          {
+            label: t('Replays'),
+            to: `/${prefix}/replays/`,
+            check: {features: 'session-replay-ui', hook: 'replay-sidebar-item'},
+          },
+          {
+            label: t('Discover'),
+            to: getDiscoverLandingUrl(organization),
+            check: {features: 'discover-basic', hook: 'discover2-sidebar-item'},
+          },
+          {label: t('Releases'), to: `/${prefix}/releases/`},
+          {label: t('Crons'), to: `/${prefix}/crons/`},
+        ],
+      },
+      {
+        label: t('Insights'),
+        icon: <IconGraph />,
+        check: {features: 'insights-entry-points'},
+        submenu: [
+          {label: MODULE_SIDEBAR_TITLE, to: `/${prefix}/${moduleURLBuilder('http')}/`},
+          {label: MODULE_TITLES.db, to: `/${prefix}/${moduleURLBuilder('db')}/`},
+          {
+            label: MODULE_TITLES.resource,
+            to: `/${prefix}/${moduleURLBuilder('resource')}/`,
+          },
+          {
+            label: MODULE_TITLES.app_start,
+            to: `/${prefix}/${moduleURLBuilder('app_start')}/`,
+          },
+          {
+            label: MODULE_TITLES['mobile-screens'],
+            to: `/${prefix}/${moduleURLBuilder('mobile-screens')}/`,
+            check: {features: 'insights-mobile-screens-module'},
+          },
+          {label: MODULE_TITLES.vital, to: `/${prefix}/${moduleURLBuilder('vital')}/`},
+          {label: MODULE_TITLES.cache, to: `/${prefix}/${moduleURLBuilder('cache')}/`},
+          {label: MODULE_TITLES.queue, to: `/${prefix}/${moduleURLBuilder('queue')}/`},
+          {
+            label: MODULE_TITLES.ai,
+            to: `/${prefix}/${moduleURLBuilder('ai')}/`,
+            check: {features: 'insights-entry-points'},
+          },
+        ],
+      },
+      {
+        label: t('Perf.'),
+        to: '/performance/',
+        icon: <IconLightning />,
+        check: {features: 'performance-view', hook: 'performance-sidebar-item'},
+      },
+      {
+        label: t('Boards'),
+        to: '/dashboards/',
+        icon: <IconDashboard />,
+        check: {
+          features: ['discover', 'discover-query', 'dashboards-basic', 'dashboards-edit'],
+          hook: 'dashboards-sidebar-item',
+        },
+      },
+      {label: t('Alerts'), to: `/${prefix}/alerts/rules/`, icon: <IconSiren />},
+      NAV_DIVIDER,
+      // {label: t('Help'), to: '', icon: <IconQuestion />},
+      // {label: t('New'), to: '', icon: <IconBroadcast />},
+      // {label: t('Stats'), to: '', icon: <IconStats />},
```

Comment:
```suggestion
```

Reply:
Removed in [92d70b5](https://github.com/getsentry/sentry/pull/77665/commits/92d70b59ab5a518fa704b5483e958590d2fbe50c).